python/cudf/cudf/io/csv.py

```diff
@@ -59,17 +59,10 @@ def read_csv(
             "`read_csv` does not yet support reading multiple files"
         )
 
-    # Only need to pass byte_ranges to get_filepath_or_buffer
-    # if `use_python_file_object=False`
-    byte_ranges = None
-    if not use_python_file_object and byte_range:
-        byte_ranges = [byte_range]
```

Comment:
This was an experimental optimization that is not used by default (and I highly doubt anyone users have manually activated it).  Now that cudf can handle pyarrow-based file objects, there is no reason to maintain this.